### PR TITLE
Eng 1278 add usage count endpoint for groups

### PR DESCRIPTION
**Pull request by Kerruba**
This introduce the `/groups/{code}/usage` endpoint to be used by the ECR to verify if a group is removable or not

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/org/entando/entando/web/group/GroupController.java [hunk 1]
```diff
@@ -85,16 +94,34 @@ public void setGroupValidator(GroupValidator groupValidator) {
...
     }
 
+    @ApiOperation("Retrieve pageModel usage count")
+    @ApiResponses({
+            @ApiResponse(code = 200, message = "OK")
+    })
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(value = "/{code}/usage", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<SimpleRestResponse<ComponentUsage>> getComponentUsage(@PathVariable String code) {
+        logger.trace("get {} usage by code {}", COMPONENT_ID, code);
```
Comment (firegloves) [inline src/main/java/org/entando/entando/web/group/GroupController.java:104]: shouldn't we use the sanitized logger added by Walter Caffiero?

**Review by firegloves — APPROVED** [review]
LGTM

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/46#pullrequestreview-490643925" title="Approved">firegloves</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/46#pullrequestreview-490643925" title="Approved">firegloves</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/46#pullrequestreview-490643925" title="Approved">firegloves</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/46#pullrequestreview-490643925" title="Approved">firegloves</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**